schema.sql

```diff
@@ -1,10 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    id INT NOT NULL,
+    id SERIAL PRIMARY KEY,
     name VARCHAR(100) NOT NULL,
     date_of_birth DATE NOT NULL,
     escape_attempts INT NOT NULL,
     neutered BOOLEAN NOT NULL,
-    weight_kg DECIMAL NOT NULL
+    weight_kg DECIMAL NOT NULL,
+    species VARCHAR(100) 
```

Comment:
- [ ] The limit here should be 255 not 100

[OPTIONAL]
- You could use ``ALTER TABLE animals ADD species varchar(255);`` to add species to the animals table.